Add info about batches retention (#11555)
DataDog/documentation: content/en/developers/faq/data-collection-resolution-retention.md

|Synthetic Monitoring|API Test results                              |Datadog Synthetic Monitoring application                                                      |User-defined         |1 min                |Result seen in the UI by user: 15 months <br/> Result not seen in the UI by user: 2 months        |
|Synthetic Monitoring|Browser Test metrics                          |Datadog Synthetic Monitoring application                                                      |User-defined         |5 min                |15 months                                                                                   |
|Synthetic Monitoring|Browser Test results                          |Datadog Synthetic Monitoring application                                                      |User-defined         |5 min                |Result seen in the UI by user: 15 months <br/> Result not seen in the UI by user: 2 months        |

> |Synthetic Monitoring|Batches                          |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci][3] or trigger API endpoints)   |Depending on calls to [@datadog/datadog-ci][3] or trigger API endpoints         |n/a                |15 days        |

Author: ```suggestion
|Synthetic Monitoring|Batches                          |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints)   |Depending on calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints         |n/a                |15 days        |
```

> |Synthetic Monitoring|Batches                          |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints)   |Depending on calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints         |n/a                |15 days        |

Author: ```suggestion
|Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints)   |Depending on calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints         |n/a                |15 days        |
```

> |Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints)   |Depending on calls to [@datadog/datadog-ci synthetics][3] or trigger API endpoints         |n/a                |15 days        |

Author: ```suggestion
|Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or [trigger API endpoint][4])   |Depending on calls to [@datadog/datadog-ci synthetics][3] or [trigger API endpoint][4]         |n/a                |15 days        |
```

> |Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [@datadog/datadog-ci synthetics][3] or [trigger API endpoint][4])   |Depending on calls to [@datadog/datadog-ci synthetics][3] or [trigger API endpoint][4]         |n/a                |15 days        |

Author: ```suggestion
|Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [Synthetic trigger API endpoint][3] or to [Synthetics CI CLI][4])   |Depending on calls to [Synthetic trigger API endpoint][3] or to [Synthetics CI CLI][4]         |n/a                |15 days        |
```

[1]: /tracing/guide/trace_sampling_and_storage/?tab=java#trace-storage
[2]: /integrations/faq/cloud-metric-delay/#faster-metrics

> [3]: https://www.npmjs.com/package/@datadog/datadog-ci

Author: ```suggestion
[3]: https://www.npmjs.com/package/@datadog/datadog-ci
[4]: /synthetics/ci/?tab=apitest#trigger-tests-endpoint
```

|Synthetic Monitoring|API Test results                              |Datadog Synthetic Monitoring application                                                      |User-defined         |1 min                |Result seen in the UI by user: 15 months <br/> Result not seen in the UI by user: 2 months        |
|Synthetic Monitoring|Browser Test metrics                          |Datadog Synthetic Monitoring application                                                      |User-defined         |5 min                |15 months                                                                                   |
|Synthetic Monitoring|Browser Test results                          |Datadog Synthetic Monitoring application                                                      |User-defined         |5 min                |Result seen in the UI by user: 15 months <br/> Result not seen in the UI by user: 2 months        |

> |Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [Synthetic trigger API endpoint][3] or to [Synthetics CI CLI][4])   |Depending on calls to [Synthetic trigger API endpoint][3] or to [Synthetics CI CLI][4]         |n/a                |15 days        |

Author: ```suggestion
|Synthetic Monitoring|Batches                                       |Datadog Synthetic Monitoring application (via calls to [Synthetic trigger API endpoint][3] or to [Synthetic CI CLI][4])   |Depending on calls to [Synthetic trigger API endpoint][3] or to [Synthetic CI CLI][4]         |n/a                |15 days        |
```

[1]: /tracing/guide/trace_sampling_and_storage/?tab=java#trace-storage
[2]: /integrations/faq/cloud-metric-delay/#faster-metrics

> [3]: https://www.npmjs.com/package/@datadog/datadog-ci

Author: ```suggestion
[3]: /synthetics/ci/?tab=apitest#trigger-tests-endpoint
```

> [4]: /synthetics/ci/?tab=apitest#trigger-tests-endpoint

Author: ```suggestion
[4]: /synthetics/ci/?tab=apitest#cli-usage
```
